arch/arm/boot/dts/xilinx/ni-zynq.dtsi

```diff
@@ -124,108 +105,6 @@
 			clock-frequency = <58824000>;
 		};
 
-		can0: can@e0008000 {
-			compatible = "xlnx,zynq-can-1.00.a";
-			status = "disabled";
-			reg = <0xe0008000 0x1000>;
-			interrupts = <0 28 4>;
-		};
-
-		can1: can@e0009000 {
-			compatible = "xlnx,zynq-can-1.00.a";
-			status = "disabled";
-			reg = <0xe0009000 0x1000>;
-			interrupts = <0 51 4>;
-		};
-
-		fpgaperipheral@f8007000 {
```

Comment:
I don't see 'fpgaperipheral' listed in the parent zynq-7000.dtsi. I guess the base address matches 'devcfg@f8007000' so removing it here may be the right thing but I can't tell since it's not listed in the commit message.

Reply:
The devcfg node does appear to represent the Xilinx FPGA manager ([ref](https://www.kernel.org/doc/Documentation/devicetree/bindings/fpga/xilinx-zynq-fpga-mgr.txt)). Will amend the commit message to clarify this.